crates/ruff_python_ast/src/nodes.rs

```diff
@@ -1482,6 +1482,12 @@ impl StringLiteralFlags {
         self
     }
 
+    #[must_use]
+    pub fn without_triple_quotes(mut self) -> Self {
+        self.0.remove(StringLiteralFlagsInner::TRIPLE_QUOTED);
+        self
+    }
```

Comment:
could we add this method to `BytesLiteralFlags`, `FStringFlags` and `AnyStringFlags` as well? We might not need it now on those structs, but it's nice to have a consistent interface between the various structs